test-runner/chrome/content/loggest-chrome-runner.js

```diff
@@ -764,14 +698,16 @@ function printTestSummary(summary) {
     str += '\x1b[0m ' + test.name + ' (' + test.variant + ')\n';
     dump(str);
 
-    // (brief) failure details:
-    if (test.result === 'fail') {
-      dump('             failing step: ' + test.firstFailedStep + '\n');
-
-      if (TEST_PARAMS.printTravisUrls && summary._filename) {
+    if (test.result === 'fail' && test.lastError) {
+      dump('             last error: ' + test.lastError + '\n');
+      if (test.lastError.stack) {
+        dump(test.lastError.stack + '\n');
+      }
+      if (TEST_PARAMS.printTravisUrls && summary.filename) {
```

Comment:
Since we no longer produce ArbPL-friendly stuff, we should probably just remove this travis/clicky stuff.  (I mean, we could make ArbPL understand, but I am very happy about the logic-inspector killing ArbPL and its wmsy-based UI, so let's not do that! ;)